specificatie/informatie-service/openapi.yaml

```diff
@@ -256,26 +256,6 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Bewoning'
-      example:
```

Comment:
waarom is dit verwijderd? Dit wordt getoond in Redocly, zodat consumers een voorbeeld hebben van de bewoning response

Reply:
Dit moest nog worden uitgebreid. De velden Leeftijd en naam.volledigeNaam zijn nu hier ook toegevoegd.